<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_25">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>姓名(必)：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="student_username_2"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_26">
           <item>
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>籍贯：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="student_home_2"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_27">
           <item>
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>电话号码：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="student_phone_2"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_11">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_28">
         <item>
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>学院：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="student_college_2"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_29">
           <item>
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>班级：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="student_class_2"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_30">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_31">
           <item>
            <widget class="QLabel" name="label_31">
             <property name="text">
              <string>学号(必)：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="student_id_2"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_13">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_32">
           <item>
            <widget class="QLabel" name="label_32">
             <property name="text">
              <string>出生年月：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="student_birth_2"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_14">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_33">
           <item>
            <widget class="QOpenGLWidget" name="openGLWidget_2"/>
           </item>
           <item>
            <widget class="QLabel" name="label_33">
             <property name="text">
              <string>邮箱：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="student_email_2"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_34">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_35">
           <item>
            <widget class="QLabel" name="label_34">
             <property name="text">
              <string>身份证号码(必)：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="student_sfzid_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_24">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_36">
           <item>
            <widget class="QLabel" name="label_35">
             <property name="text">
              <string>家庭居住地：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="student_live_2"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_15">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_37">
           <item>
            <widget class="QLabel" name="label_36">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>政治面貌：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="student_identity_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_38">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_71">
           <item>
            <widget class="QLabel" name="label_62">
             <property name="text">
              <string>家庭月收入（元）：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="student_income_2"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_16">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_39">
           <item>
            <widget class="QLabel" name="label_37">
             <property name="text">
              <string>贫困类型</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_pinkun_2">
             <item>
              <property name="text">
               <string>无</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>低保</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>残障</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>患病</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>单亲</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>其他</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_17">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_40">
           <item>
            <widget class="QLabel" name="label_38">
             <property name="text">
              <string>家庭户口类型：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_hukou_2">
             <item>
              <property name="text">
               <string>无</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>城镇</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>农村</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>照片</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_39">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../../../.designer/img/人脸识别.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_12">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="butto_sure">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
